Controllers/ReviewController.cs

```diff
@@ -97,6 +97,91 @@ public IHttpActionResult GetAllReviews()
             return Ok(reviews);
         }
 
+
+        // ===================================================================================================
+        [HttpGet]
+        [Route("~/api/user/{userId:int}/reviews")]
+        public IHttpActionResult GetUserReviews(int userId)
+        {
+            Review review = null;
+
+            try
+            {
+                SqlConnection sqlConnection = new SqlConnection(DB_CONNECTION_STRING);
+                using (sqlConnection)
+                {
+                    SqlCommand selectReviewById = new SqlCommand(@"SELECT * FROM Review
+                                                                    WHERE Use_ID = @Use_ID", sqlConnection);
+                    selectReviewById.Parameters.AddWithValue("Use_ID", userId);
+                    sqlConnection.Open();
+                    SqlDataReader sqlDataReader = selectReviewById.ExecuteReader();
+                    while (sqlDataReader.Read())
+                    {
+                        review = new Review
+                        {
+                            ID = sqlDataReader.GetInt32(0),
+                            Date = sqlDataReader.GetDateTime(1),
+                            Rate = sqlDataReader.GetInt32(2),
+                            Recommendation = sqlDataReader.GetBoolean(3),
+                            Comment = sqlDataReader.GetString(4),
+                            Usefull = sqlDataReader.GetInt32(5),
+                            Title = sqlDataReader.GetString(6),
+                            UserID = sqlDataReader.GetInt32(7),
+                            AccomodationID = sqlDataReader.GetInt32(8)
+                        };
+                    }
+                }
+
+            }
+            catch (Exception ex)
+            {
+                return InternalServerError(ex);
+            }
+            return Ok(review);
+        }
+
+        // ===================================================================================================
+        [HttpGet]
+        [Route("~/api/accomodation/{accomodationId:int}/reviews")]
+        public IHttpActionResult GetAccomodationReviews(int accomodationId)
+        {
+            Review review = null;
+
+            try
+            {
+                SqlConnection sqlConnection = new SqlConnection(DB_CONNECTION_STRING);
+                using (sqlConnection)
+                {
+                    SqlCommand selectReviewById = new SqlCommand(@"SELECT * FROM Review
```

Comment:
Give names of column instead of "*" for better query performance:
SELECT [Rev_ID]
,[Rev_Date]
,[Rev_Rate]
,[Rev_Recommendation]
,[Rev_Comment]
,[Rev_Usefull]
,[Rev_Title]
,[Use_ID]
,[Acc_ID]
FROM [dbo].[Review]

Reply:
good catch thanks!